paddle/fluid/operators/math/selected_rows_functor.cc

```diff
@@ -195,6 +195,10 @@ struct SelectedRowsAddToTensor<platform::CPUDeviceContext, T> {
   void operator()(const platform::CPUDeviceContext& context,
                   const framework::SelectedRows& input1,
                   framework::Tensor* input2) {
+    if (input1.rows().size() == 0) {
```

Comment:
`if (UNLIKELY())` should boost the performance

Reply:
done